src/index.html

```diff
@@ -14,15 +14,15 @@
 	</style>
 	<div class="container">
 		<header>
-			<h2>We're live at <a class="mklive" href="live">milkenknights.com/live</h2></a>
-			<p><b>Las Vegas Regional</b>
-				&emsp;&emsp;&emsp;3.27.19 - 3.30.19
-				&emsp;&emsp;&emsp;Las Vegas, NV</p>
+			<a class="mklive" href="live"><h2>We're live at milkenknights.com/live</h2></a>
+			<p><b>Los Angeles Regional</b>
+				&emsp;&emsp;&emsp;3.5.19 - 3.8.19
+				&emsp;&emsp;&emsp;Los Angeles, CA</p>
 	</div>
 	</header>
 	</div>
-</section> -->
-<!--
+</section>
+
```

Comment:
You should _not_ have gotten rid of the starting comment tags here. This is a CTA from the 2018 robot. Add `<!--` back in on this line.